src/interfaces/responses.ts

```diff
@@ -0,0 +1,11 @@
+interface BaseResponse {
+  allowed: boolean;
+}
+
+interface TokenBucketStrategyResponse {
+  allowed: boolean;
+  remaining: number;
+  retryAfter: number; // in milliseconds
```

Comment:
```suggestion
  retryAfterInMs: number; // in milliseconds
```
More meaningful name